airbyte-integrations/connectors/source-zoom/integration_tests/expected_records.jsonl

```diff
@@ -0,0 +1,15 @@
participant_video": false, "cn_meeting": false, "in_meeting": false, "join_before_host": true, "jbh_time": 0, "mute_upon_entry": true, "watermark": false, "use_pmi": false, "approval_type": 0, "audio": "both", "auto_recording": "none", "enforce_login": false, "enforce_login_domains": "", "alternative_hosts": "", "alternative_host_update_polls": false, "close_registration": false, "show_share_button": true, "allow_multiple_devices": true, "registrants_confirmation_email": true, "waiting_room": false, "request_permission_to_unmute_participants": false, "global_dial_in_countries": ["US"], "global_dial_in_numbers": [{"country_name": "US", "number": "+1 646 931 3860", "type": "toll", "country": "US"}, {"country_name": "US", "city": "New York", "number": "+1 929 436 2866", "type": "toll", "country": "US"}, {"country_name": "US", "city": "Washington DC", "number": "+1 301 715 8592", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 305 224 1968", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 309 205 3325", "type": "toll", "country": "US"}, {"country_name": "US", "city": "Chicago", "number": "+1 312 626 6799", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 719 359 4580", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 253 205 0468", "type": "toll", "country": "US"}, {"country_name": "US", "city": "Tacoma", "number": "+1 253 215 8782", "type": "toll", "country": "US"}, {"country_name": "US", "city": "Houston", "number": "+1 346 248 7799", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 360 209 5623", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 386 347 5053", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 507 473 4847", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 564 217 2000", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 669 444 9171", "type": "toll", "country": "US"}, {"country_name": "US", "city": "San Jose", "number": "+1 669 900 6833", "type": "toll", "country": "US"}, {"country_name": "US", "number": "+1 689 278 1000", "type": "toll", "country": "US"}], "registrants_email_notification": true, "meeting_authentication": false, "encryption_type": "enhanced_encryption", "approved_or_denied_countries_or_regions": {"enable": false}, "breakout_room": {"enable": false}, "internal_meeting": false, "continuous_meeting_chat": {"enable": false, "auto_add_invited_external_users": false}, "participant_focused_meeting": false, "push_change_to_calendar": false, "resources": [], "alternative_hosts_email_notification": true, "show_join_info": true, "device_testing": false, "focus_mode": false, "enable_dedicated_group_chat": false, "private_meeting": false, "email_notification": true, "host_save_video_order": false, "sign_language_interpretation": {"enable": false}, "email_in_attendee_report": false}, "pre_schedule": false}, "emitted_at": 1709070638055}
```

Comment:
Does this contain PII ?

I'm checking the `password` and other properties, it seems like it is. @katmarkham @ChristoGrab

Reply:
I'm pretty sure the passwords are harmless since they're just passwords to join dummy Zoom meetings in the sandbox account, but I've removed all `password` related keys from the expected records just in case to be sure.